### PR TITLE
fill( some js functions )

### PR DESCRIPTION
**Pull request by izio38**
This PR fill some js functions:

- emit
- emitNet
- on
- onNet
- exports

### CODE REVIEW

File: content/scripting-reference/runtimes/javascript/functions/emit-client.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Emit - client
+---
+
+Use `emit` function when you want to trigger an even from the client to the client.\
```
Comment (d0p3t) [inline content/scripting-reference/runtimes/javascript/functions/emit-client.md]: Spelling mistake - `even` -> `event`.

File: content/scripting-reference/runtimes/javascript/functions/emit-client.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+You need to declare the event on the server with [on](/scripting-reference/runtimes/javascript/functions/on-client) or [onNet](/scripting-reference/runtimes/javascript/functions/onNet-client) first.
```
Comment (d0p3t) [inline content/scripting-reference/runtimes/javascript/functions/emit-client.md]: `on the server` - I'm guessing you meant `on the client`

File: content/scripting-reference/runtimes/javascript/functions/emit-server.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+---
+title: Emit - server
+---
+
+Use `emit` function when you want to trigger an even from the server to the server.\
```
Comment (d0p3t) [inline content/scripting-reference/runtimes/javascript/functions/emit-server.md]: Same as client

File: content/scripting-reference/runtimes/javascript/functions/emitNet-client.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
+title: EmitNet - client
+---
+
+Use `emitNet` function when you want to trigger an even from the client to the server.\
```
Comment (d0p3t) [inline content/scripting-reference/runtimes/javascript/functions/emitNet-client.md]: Again spelling mistake

File: content/scripting-reference/runtimes/javascript/functions/emitNet-server.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+---
+title: EmitNet - server
+---
+
+Use `emitNet` function when you want to trigger an even from the server to a client or to all clients.\
```
Comment (d0p3t) [inline content/scripting-reference/runtimes/javascript/functions/emitNet-server.md]: And again :)

File: content/scripting-reference/runtimes/javascript/functions/exports.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+---
+title: Exports
+---
+
+Use this to export functions so they can be called from other resources.\
+Read more on [exports](http://localhost:1313/scripting-reference/resource-manifest/resource-manifest/#export)
```
Comment (d0p3t) [inline content/scripting-reference/runtimes/javascript/functions/exports.md]: Wrong link url

File: content/scripting-reference/runtimes/javascript/functions/on-client.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
+---
+title: On - client
+---
+
+Use `on` function when you want to listen from client only.\
+This declare an event which get triggered with [emit](/scripting-reference/runtimes/javascript/functions/emit-client) from the client.
```
Comment (d0p3t) [inline content/scripting-reference/runtimes/javascript/functions/on-client.md]: `declares`

File: content/scripting-reference/runtimes/javascript/functions/on-server.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
+---
+title: On - server
+---
+
+Use `on` function when you want to listen from server only.\
+This declare an event which get triggered with [emit](/scripting-reference/runtimes/javascript/functions/emit-server) from the server.
```
Comment (d0p3t) [inline content/scripting-reference/runtimes/javascript/functions/on-server.md]: Same as client

File: content/scripting-reference/runtimes/javascript/functions/onNet-client.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+---
+title: onNet - client
+---
+
+Use `onNet` function when you want to listen from client **AND** from server.\
+This declare an event which get triggered with [emit](/scripting-reference/runtimes/javascript/functions/emit-client) (from client) or [emitNet](/scripting-reference/runtimes/javascript/functions/emitNet-server) (from server).
```
Comment (d0p3t) [inline content/scripting-reference/runtimes/javascript/functions/onNet-client.md]: Again `declares` and `gets` :)

File: content/scripting-reference/runtimes/javascript/functions/onNet-server.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
+---
+title: onNet - server
+---
+
+Use `onNet` function when you want to listen from server **AND** from client.\
+This declare an event which get triggered with [emit](/scripting-reference/runtimes/javascript/functions/emit-server) (from server) or [emitNet](/scripting-reference/runtimes/javascript/functions/emitNet-client) (from client).
```
Comment (d0p3t) [inline content/scripting-reference/runtimes/javascript/functions/onNet-server.md]: Same as previously

File: content/scripting-reference/runtimes/javascript/functions/exports.md (hunk 6 above)
Comment (izio38) [reply content/scripting-reference/runtimes/javascript/functions/exports.md]: What do you suggest?

**Pull request merged**